internal/clients/team3/utilfunctions.go

```diff
@@ -131,3 +130,16 @@ func leastTrusted(values map[shared.ClientID]float64) shared.ClientID {
 	}
 	return leastTrustedClient
 }
+
+// getIslandsAlive retrives number of islands still alive
+func (c *client) isClientStatusCritical(ClientID shared.ClientID) bool {
```

Comment:
could shorten to:
`return c.BaseClient.ServerReadHandle.GetGameState().ClientLifeStatuses[ClientID] == shared.Critical`